ofrak_core/test_ofrak/unit/test_ofrak_server.py

```diff
@@ -664,3 +676,7 @@ async def test_clear_action_queue(ofrak_client: TestClient, hello_world_elf):
         "    ofrak.run(main)",
         "",
     ]
+
+    expected_str = normalize(expected_list)
```

Comment:
Does the expected string need to be normalized?

Reply:
Resolving this for same reason as above